.env

```diff
@@ -1,3 +1,3 @@
 PORT=5000
 POSTGRES_PASSWORD=password
-REDIS_URL=redis://redis:6379
\ No newline at end of file
+REDIS_URL=redis://redis:6379
```

Comment:
abrichr 9 days ago Member
Why is it necessary to add this here now if RQ was able to connect without it before? Can we find out how RQ is getting the connection string and do it the same way? (My concern is having more than one place where we have to update connection information.)

Reply:
This is the only place where I am specifying the REDIS url. In worker.py I am using this specific URL... I am not sure what you mean by RQ was able to connect without it before, it always needed the REDIS URL

Reply:
Prior to this PR, RQ was connecting to Redis. In this PR, you are adding this line. How was RQ connecting to Redis without this line?

Reply:
All that was added is a newline character, not the line itself. staging is a direct copy of master, and you can see that in master, this line is present in the .env file.